packages/frontend/.eslintrc.json

```diff
@@ -1,6 +1,9 @@
 {
-  "extends": ["next/core-web-vitals", "plugin:prettier/recommended"],
+  "parser": "@typescript-eslint/parser",
+  "extends": ["next/core-web-vitals", "plugin:prettier/recommended", "plugin:@typescript-eslint/recommended"],
   "rules": {
-    "no-unused-vars": "error"
+    "no-unused-vars": "off",
```

Comment:
I think we don't need this rule. Can you double check?

I removed that line, added this:

```
interface test {
  onChange?: (arg: string) => void;
}
```

And it works!